packages/vue-client/src/components/boards/FractionalBoard.vue

```diff
@@ -0,0 +1,108 @@
+<script setup lang="ts">
+import type {
+  FractionalConfig,
+  FractionalState,
+} from "@ogfcommunity/variants-shared";
+import { computed } from "vue";
+import TaegeukStone from "../TaegeukStone.vue";
+
+const props = defineProps<{
+  config: FractionalConfig;
+  gamestate: FractionalState;
+}>();
+
+const emit = defineEmits<{
+  (e: "move", Identifier: number): void;
+}>();
+
+function intersectionClicked(identifier: number) {
+  emit("move", identifier);
+}
+
+const viewBox = computed(() => {
+  const xPositions = props.gamestate.intersections.map((i) => i.position.X);
+  const yPositions = props.gamestate.intersections.map((i) => i.position.Y);
+  return `${Math.min(...xPositions) - 1} ${Math.min(...yPositions) - 1} ${
+    Math.max(...xPositions) + 2
+  } ${Math.max(...yPositions) + 2}`;
+});
+
+const stagedMove = computed(() => props.gamestate.stagedMove);
+</script>
+
+<template>
+  <div>
+    <h3 style="color: red">Experimental</h3>
+    <svg
+      class="board"
+      xmlns="http://www.w3.org/2000/svg"
+      width="600px"
+      height="600px"
```

Comment:
```suggestion
      width="100%"
      height="100%"
```

Let the framework handle the sizing; otherwise the home page gets a little funky 🤪

<img width="348" alt="Screenshot 2023-07-09 at 1 46 16 PM" src="https://github.com/govariantsteam/govariants/assets/25233703/3a11c401-e916-47e9-852a-a9927b556bdd">